Example/Example/ViewController.swift

```diff
@@ -657,8 +657,8 @@ class NativeEventFormViewController : FormViewController {
                         endRow.updateCell()
                     }
                 }
-                .onExpandInlineRow { cell, row, inlineRow in
-                    inlineRow.cellUpdate { [weak self] cell, dateRow in
+                .onExpandInlineRow { [weak self] cell, row, inlineRow in
+                    inlineRow.cellUpdate() { cell, row in
```

Comment:
we don't need the `()` after cellUpdate method. Anyway let's keep this code anyway.